code/game/machinery/computer/buildandrepair.dm

```diff
@@ -54,6 +54,10 @@
 /obj/item/weapon/circuitboard/med_data
 	name = "Circuit board (Medical Records)"
 	build_path = /obj/machinery/computer/med_data
+/obj/item/weapon/circuitboard/pandemic

```

Comment:
I guess I can cover this in a follow up PR---it's pretty easy though; it's just under modules->research->designs.